librubyfmt/src/format_prism.rs

```diff
@@ -235,6 +242,184 @@ fn format_statements(ps: &mut dyn ConcreteParserState, statements_node: prism::S
     );
 }
 
+fn format_string_node(ps: &mut dyn ConcreteParserState, string_node: prism::StringNode) {
+    if ps.at_start_of_line() {
+        ps.emit_indent();
+    }
```

Comment:
I thought we were doing all of these at the start of `format_node` nowadays, so we can delete this bit and the matching newline code in this function and elsewhere?